### PR TITLE
Wip/bundle adjustment

### PR DESCRIPTION
**Pull request by hannalee2**
Major Updates:

1. **Bundle Adjustment** implemented using optimization from the SciPy library (TODO: Increase precision).
    - usage: python -m parallax **-b**
2. **Reticle detection** now uses tick marks from -10 to 10 (previously: -15 to 15) for better reticle detection. 
3. **Bug Fix**:
    - Problem: Selecting the positive x-axis gave an incorrect reprojection point.
    - Fix: Updated the camera intrinsic model after accepting the positive x-axis input from the user.

### CODE REVIEW

File: parallax/bundle_adjustmnet.py
Comment (jsiegle) [inline parallax/bundle_adjustmnet.py:1]: Fix typo in file name

File: pyproject.toml [hunk 1]
```diff
@@ -26,7 +26,8 @@ dependencies = [
     "pandas",
     "scikit-image",
     "requests",
-    "scikit-learn"
+    "scikit-learn",
+    "pyceres"
```
Comment (jsiegle) [inline pyproject.toml:30]: Is `pyceres` dependency still needed?

Comment (hannalee2) [reply pyproject.toml]: Removed pyceres from the dependencies.

File: parallax/bundle_adjustmnet.py
Comment (hannalee2) [reply parallax/bundle_adjustmnet.py:1]: Fixed typo. 
Thanks for catching this!

**Pull request merged**